pkg/definition/get.go

```diff
@@ -33,7 +33,7 @@ func GetDefinition(c *client.MassdriverClient, definitionType string) (map[strin
 
 	if resp.Status != "200 OK" {
```

Comment:
```suggestion
	if resp.StatusCode != http.StatusOk {
```

Let's get this updated while you're in there.

Reply:
Ooo great point. I'll comb for any others as well